src/main/kotlin/eu/europa/ec/eudi/sdjwt/vc/SdJwtVcVerifier.kt

```diff
@@ -67,18 +67,18 @@ fun interface LookupPublicKeysFromDIDDocument {
 interface SdJwtVcVerifier<out JWT> {
 
     /**
-     * Verifies an SD-JWT (in non enveloped, simple format)
-     * Typically, this is useful to Holder that wants to verify an issued SD-JWT
+     * Verifies an SD-JWT serialized using compact serialization.
+     * Typically, this is useful to Holder that wants to verify an issued SD-JWT.
      *
      * @param unverifiedSdJwt the SD-JWT to be verified
      * @return the verified SD-JWT, if valid. Otherwise, method could raise a [SdJwtVerificationException]
      * The verified SD-JWT will contain a [JWT][SdJwt.jwt] as both string and decoded payload
      */
-    suspend fun verifyIssuance(unverifiedSdJwt: String): Result<SdJwt<JWT>>
+    suspend fun verify(unverifiedSdJwt: String): Result<SdJwt<JWT>>
 
     /**
-     * Verifies an SD-JWT in JWS JSON general of flattened format as defined by RFC7515 and extended by SD-JWT
-     * specification
+     * Verifies an SD-JWT serialized using JWS JSON serialization (either general or flattened format) as defined by RFC7515
```

Comment:
Typically, this is useful to Holder that wants to verify an issued SD-JWT or to Verifier
that wants to verify a presented SD-JWT in case the KB-JWT [must not be present]KeyBindingVerifier.MustNotBePresent]